core/tests/protractor_utils/forms.js

```diff
@@ -407,21 +442,26 @@ var MultiSelectEditor = function(elem) {
     },
     expectCurrentSelectionToBe: async function(expectedCurrentSelection) {
       // Open the dropdown menu.
-      await elem.element(by.css(
-        '.protractor-test-search-bar-dropdown-toggle')).click();
+      var SearchBarDropdownToggle = elem.element(by.css(
+        '.protractor-test-search-bar-dropdown-toggle'));
+      await action.click(
+        'Search Bar Dropdown Toggle', SearchBarDropdownToggle);
 
       // Find the selected elements.
       var actualSelection = await elem.element(
         by.css('.protractor-test-search-bar-dropdown-menu')
       ).all(by.css('.protractor-test-selected'))
         .map(async function(selectedElem) {
-          return await selectedElem.getText();
+          return await action.getText(
+            `Search bar dropdown menu ${selectedElem}`, selectedElem);
```

Comment:
What kind of string does this produce? I don't know how elements get stringified. If the string isn't helpful, you could use a for loop instead to keep track of the index

Reply:
I have renamed it as 'selected element', I think it should be sufficient. done!